spec/models/fae/static_page_spec.rb

```diff
@@ -34,18 +34,18 @@
       expect(ap.header_image_zh).to be_nil
     end
 
-    # TODO - AboutUsPage.instance instantiations in other tests seem to override the I18n locale set here, so using a one-off page. Not sure if rspec or Fae bug
-    it 'should use _content to query the field with the desired I18n locale' do
-      I18n.locale = :zh
+    # TODO - This test fails occaisionally, and those failures appear to be related to test order (likely related to changing I18n.locale). Temporarily commented out.
```

Comment:
@jamesmk See note here. This is something to revisit, but since `I18n.locale` is almost always set on the request/controller side I think we're fine without the test for now.